module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -853,6 +853,19 @@ protected function getJournalOpenUrlParams()
         return $params;
     }
 
+    /**
+     * Get an alternate display title.
+     *
+     * @return string
+     */
+    public function getAlternateDisplayTitle()
+    {
+        $format = '%s %s';
+        $short = $this->fields['alt_title_short'] ?? '';
+        $sub = $this->fields['alt_title_sub'] ?? '';
+        return sprintf($short, $sub);
+    }
+
```

Comment:
I did this for convenience but maybe this should be split out into two functions like the corresponding "normal" fields are? Or maybe the corresponding fields could be combined into 1 function, e.g. `getDisplayTitle`? I wasn't sure what to do here so I just did something quick and easy. I'm not sure if it's problematic though.

Reply:
Right now, you're indexing the short + sub titles in the `alt_title` field, so why do this concatenation at all? Wouldn't it be better to simply return the `alt_title` field directly?

Reply:
Ah yes, I didn't explain this very well. I agree with not concatenating at all. This was my attempt to keep these fields analogous with their linked counterparts. This is the title that displays on the full record page and the "normal" versions of these fields are concatenated in the templates there. I was just being lazy and doing it here, rather than in the templates because I didn't want to write two redundant methods until I understood better the direction we were going to take. If we just use `alt_title` we would be treating the 880 title that displays on the full record slightly different than the analogous title that displays. Of course this would be mitigated by the fact that they use the same subfields (in this case) and people can index them however they want, but I thought there might be a reason for that separation in the analogous fields that might also pertain to 880 titles in how they might be used at other institutions, if that makes sense.  That being said though, this is a good example of some of the weirdness that can come up here.

Reply:
My point is just that there are three record driver methods for regular titles: getTitle(), getShortTitle(), and getSubtitle(). Each of these methods corresponds to a field in the Solr index. If you want to create three parallel versions for 880's, it would make sense for each of those to also correspond with the equivalent Solr fields... and the concatenation you're doing here takes the fields I would expect you to use in ShortTitle and Subtitle and mashing them together in place of just using the Title field. It just seems like a confusingly mixed approach, though I understand that we may not want to go all-out in creating all these variations. Certainly worth discussing further before taking lots of action, especially with @EreMaijala's cautionary note. :-)